docs/ABOUT.md

```diff
@@ -16,9 +16,9 @@ The primary concepts baked into the making of the language were:
 
 Dart is a very versitile language that can be used in a variety of different ways:
 
- - Dart code can be transpiled into JavaScript for [web development](https://dart.dev/web), typically using [AngularDart](https://webdev.dartlang.org/angular).
+ - Dart code can be transpiled into JavaScript for [web development](https://dart.dev/web), typically using [AngularDart](https://github.com/angulardart).
```

Comment:
I would suggest the github link to the AngularDart package repository. They have still released new versions.

https://github.com/angulardart/angular

Reply:
I followed the original link https://webdev.dartlang.org/angular, which was redirected to https://github.com/angulardart which is why I chose this link. If you think linking directly to angular is better, I'll change the link.

Reply:
I do think it would be better.

Reply:
Done ☺️